src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -0,0 +1,369 @@
+package org.carlmontrobotics.robotcode2023.subsystems;
+
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.*;
+
+import org.carlmontrobotics.MotorConfig;
+import org.carlmontrobotics.lib199.MotorControllerFactory;
+import org.carlmontrobotics.robotcode2023.Constants.GoalPos;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxAbsoluteEncoder;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+// Arm angle is measured from horizontal on the intake side of the robot and bounded between -3π/2 and π/2
+// Wrist angle is measured relative to the arm with 0 being parallel to the arm and bounded between -π and π (Center of Mass of Roller)
+public class Arm extends SubsystemBase {
+
+    private final CANSparkMax armMotor = MotorControllerFactory.createSparkMax(armMotorPort, MotorConfig.NEO);
+    private final CANSparkMax wristMotor = MotorControllerFactory.createSparkMax(wristMotorPort, MotorConfig.NEO);
+    private final SparkMaxAbsoluteEncoder armEncoder = armMotor
+            .getAbsoluteEncoder(SparkMaxAbsoluteEncoder.Type.kDutyCycle);
+    private final SparkMaxAbsoluteEncoder wristEncoder = wristMotor
+            .getAbsoluteEncoder(SparkMaxAbsoluteEncoder.Type.kDutyCycle);
+
+    private final SimpleMotorFeedforward armFeed = new SimpleMotorFeedforward(kS[ARM], kV[ARM], kA[ARM]);
+    private ArmFeedforward wristFeed = new ArmFeedforward(kS[WRIST], kG[WRIST], kV[WRIST], kA[WRIST]);
+
+    public PIDController armPID = new PIDController(kP[ARM], kI[ARM], kD[ARM]);
+    public PIDController wristPID = new PIDController(kP[WRIST], kI[WRIST], kD[WRIST]);
+
+    private SendableBuilder senb;
+
+    public int object = CUBE;
+    public boolean isFront = true;
+
+    public Arm() {
+        armMotor.setInverted(inverted[ARM]);
+        wristMotor.setInverted(inverted[WRIST]);
+
+        armEncoder.setPositionConversionFactor(rotationToRad);
+        wristEncoder.setPositionConversionFactor(rotationToRad);
+
+        wristEncoder.setZeroOffset(offsetRad[ARM]);
+        armEncoder.setZeroOffset(offsetRad[WRIST]);
+
+        armPID.setTolerance(posToleranceRad[ARM], velToleranceRadPSec[ARM]);
+        wristPID.setTolerance(posToleranceRad[WRIST], velToleranceRadPSec[WRIST]);
+
+        SmartDashboard.putBoolean("Toggle", false);
```

Comment:
Not clear what this is for.

Note that you probably want an "Apply Risky Changes" button that, when clicked, sets the values that you don't want set with every keystroke in Shuffleboard. The setters for those properties would set an intermediate variable and the "Apply Risky Changes" would copy the values from those variables into the real variables.